doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1305,22 +1305,22 @@ Create a Private URL (must be able to manage dataset permissions):
 
   export API_TOKEN=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
   export SERVER_URL=https://demo.dataverse.org
-  export ID=24
+  export PERSISTENT_IDENTIFIER=doi:10.5072/FK2/J8SJZB
 
-  curl -H "X-Dataverse-key: $API_TOKEN" -X POST $SERVER_URL/api/datasets/$ID/privateUrl
+  curl -H "X-Dataverse-key: $API_TOKEN" -X POST $SERVER_URL/api/datasets/:persistentId/privateUrl?persistentId=$PERSISTENT_IDENTIFIER
```

Comment:
For this and the others where `ID` was changed to `PERSISTENT_IDENTIFIER` I think there's some confusion.

`ID` should work but it's the database ID of the dataset and not especially easy to discover. (You can find it in the JSON export.)

The history on this that at first only database IDs where supported. Then we created #1837 to add DOI support (`PERSISTENT_IDENTIFIER`). Then DOI support was added in PR #2893. But the old docs with `ID` were not changed. And again, `ID` should work. Both should work.

I think it's better to switch to `PERSISTENT_IDENTIFIER`, honestly. The DOI is what people easily see! So I think this PR is good step in the right direction. We just just clarify somewhere the both work.